dask_kubernetes/core.py

```diff
@@ -415,6 +426,8 @@ def __init__(
         security=None,
         scheduler_service_wait_timeout=None,
         scheduler_pod_template=None,
+        scheduler_service_type="ClusterIP",
```

Comment:
This should be set to `None` as the default is already set in the configuration YAML.